test/smoke_test/smoke_test.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import torch
+import torchvision
+import torchaudio
+import torch.nn as nn
+import torchaudio.compliance.kaldi  # noqa: F401
+import torchaudio.datasets  # noqa: F401
+import torchaudio.functional  # noqa: F401
+import torchaudio.models  # noqa: F401
+import torchaudio.pipelines  # noqa: F401
+import torchaudio.sox_effects  # noqa: F401
+import torchaudio.transforms  # noqa: F401
+import torchaudio.utils  # noqa: F401
+from torchaudio.io import StreamReader
+import torchvision.datasets as dset
+import torchvision.transforms
+cuda_version_expected = os.environ['CUDA_VER']
+is_cuda_system = cuda_version_expected != "cpu"
+#todo add torch, torchvision and torchaudio tests
+print(torch.__version__)
+print(torchvision.__version__)
+print(torchaudio.__version__)
+print('Is torchvision useable?', all(x is not None for x in [torch.ops.image.decode_png, torch.ops.torchvision.roi_align]))
+if(not torch.cuda.is_available() and is_cuda_system):
+    print(f"Expected CUDA {cuda_version_expected}. However CUDA is not loaded.")
+    sys.exit(1)
+if(torch.cuda.is_available()):
```

Comment:
nit: outer parentheses are not necessary in `if`

Reply:
I think its better to use parentheses in all if statements for consistency

Reply:
I checked other python files and they use the style without parentheses, but, since we don't enforce a common style in the builder project (yet?), I think it's ok, we can always reformat later, if we have the need.